common/get-host-info.sub.js

```diff
@@ -0,0 +1,19 @@
+function get_host_info() {
+
+  var HTTP_PORT = eval('{{ports[http][0]}}');
+  var HTTP_PORT2 = eval('{{ports[http][1]}}');
+  var HTTPS_PORT = eval('{{ports[https][0]}}');
+  var ORIGINAL_HOST = eval('\'{{host}}\'');
```

Comment:
Why eval here (and above)? Can't it be
```
var HTTP_PORT = {{ports[http][0]}};
```
etc (if you want it as a JS number, but maybe it should be a string instead?)?